src/Field.tsx

```diff
@@ -171,11 +169,23 @@ class FieldInner<Values = {}, Props = {}> extends React.Component<
     }
 
     if (typeof component === 'string') {
-      const { innerRef, ...rest } = props;
+      const { innerRef, ...restProps } = props;
+      const { value, ...restField } = field;
+      const valueProp: { checked?: boolean; value?: typeof value } = {};
+
+      if (props.type === 'radio') {
```

Comment:
Probably ways to simplify with an es6 computed property.

{ [type === radio ? ‘checked’ : ...]: getIn... }

Reply:
Full implementation then would look something like:
```js
const useChecked = type === 'radio' || type === 'checkbox'
{ [useChecked ? 'checked' : 'value']: type === 'radio' ? props.value === value : value
```

do you think it's better?